src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx

```diff
@@ -15,17 +15,14 @@ const CrumbTable = React.createClass({
     if (!this.props.kvData) {
       return null;
     }
-    return _.chain(this.props.kvData)
-      .map((val, key) => [val, key])
-      .map(([val, key]) => {
-        return (
-          <tr key={key}>
-            <td className="key">{key}</td>
-            <td className="value"><pre>{val + ''}</pre></td>
-          </tr>
-        );
-      })
-      .value();
+    return _.map(this.props.kvData, (val, key) => [val, key]).map(([val, key]) => {
+      return (
```

Comment:
`return _.map(this.props.kvData, (val, key) => [val, key]).map(([val, key]) => {`
is equivalent to
`return _.map(this.props.kvData, (val, key) => {`